map_model/src/pathfind/v1.rs

```diff
@@ -675,11 +686,13 @@ fn validate_continuity(map: &Map, steps: &[PathStep]) {
             PathStep::Lane(id) => map.get_l(id).last_pt(),
             PathStep::ContraflowLane(id) => map.get_l(id).first_pt(),
             PathStep::Turn(id) => map.get_t(id).geom.last_pt(),
+            PathStep::ContraflowTurn(id) => map.get_t(id).geom.first_pt(),
         };
         let to = match pair[1] {
             PathStep::Lane(id) => map.get_l(id).first_pt(),
             PathStep::ContraflowLane(id) => map.get_l(id).last_pt(),
             PathStep::Turn(id) => map.get_t(id).geom.first_pt(),
+            PathStep::ContraflowTurn(id) => map.get_t(id).geom.first_pt(),
```

Comment:
Should be `last_pt`, right? This validation is commented out, could be good to enable it to double-check